cloudferrylib/os/image/glance_image.py

```diff
@@ -190,10 +190,10 @@ def image_exists(self, image_id):
             except glance_exceptions.HTTPNotFound:
                 return False
 
-    def get_image_list(self):
+    def get_image_list(self, is_filtering=True):
         images = self.glance_client.images.list(filters={"is_public": None})
```

Comment:
It's exactly our case: without context, what this call could mean `glance_image.get_image_list(False)`? How is it different from `glance_image.get_image_list(True)`? To make it obvious you will ned to rename function to something like `get_filtered_or_image_list_if_first_arg_true_non_filtered_otherwise(True)`, but it would be insane.